### DISCUSSION TITLE
How to integrate OTA and MQTT credentials features in one sketch

### DISCUSSION THREAD

**Discussion by TutorKidsDad** [Hieromon/AutoConnect#398] [How can I?] [answered]
Hi I am working on a sketch to integrate OTA facility and updating Thingspeak MQTT credentials features in one sketch through a single captive portal ,no luck till now.any help is appreciated.

**Comment by Hieromon** [comment, marked as answer]
It allows enabling OTA using AutoConnectConfig::ota setting. This setting works even when the ESP module is not connected to any access point. (that is, the captive portal has open)
See the [documentation](https://hieromon.github.io/AutoConnect/otabrowser.html#how-to-embed-autoconnectota-in-your-sketch).

**Comment by TutorKidsDad** [comment]
Thanks for the prompt reply,I'll chk and revert back to you as soon as
possible
